zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/changes/PartitionJoinApplier.java

```diff
@@ -125,6 +128,18 @@ public ActorFuture<UnaryOperator<MemberState>> applyOperation() {
     return result;
   }
 
+  private DynamicPartitionConfig getPartitionConfig(
+      final ClusterConfiguration currentClusterConfiguration) {
+    // Find configuration from any other member that has the same partition. We can assume that
```

Comment:
:thought_balloon: It's also only correct because configuration is not being changed at the same time as scaling :sweat_smile:

Reply:
Yes. I don't know how else we can guarantee a consistent state for exporters.